js/tabs.js

```diff
@@ -0,0 +1,42 @@
+var tabs = document.querySelectorAll('.js-tab');
+
+function setupTabs() {
+  for (var i = 0; i < tabs.length; i += 1) {
+    var tab = tabs[i],
+        tabSelector = tab.getAttribute('href'),
+        tabContent = document.querySelector(tabSelector);
+
+    if (tabContent) {
+      tab.setAttribute('aria-selected', 'false');
+      tabContent.classList.add('u-hide');
+
+      tab.addEventListener('click', function(e) {
+        e.preventDefault();
+        toggleTab(event.currentTarget);
+      });
+    }
+  }
+}
+
+function toggleTab(tab) {
+  var tabSelector = tab.getAttribute('href'),
+      tabContent = document.querySelector(tabSelector);
+
+  for (var i = 0; i < tabs.length; i += 1) {
+    if (tabs[i] !== tab) {
+      var otherTabSelector = tabs[i].getAttribute('href'),
+          otherTabContent = document.querySelector(otherTabSelector);
+      
+      tabs[i].setAttribute('aria-selected', 'false');
+      tabs[i].classList.remove('active');
+      otherTabContent.classList.add('u-hide');
+    }
+  }
+
+  tab.setAttribute('aria-selected', 'true');
+  tab.classList.toggle('active');
+  tabContent.classList.toggle('u-hide');
+  tabContent.scrollIntoView({ behavior: 'smooth' });
```

Comment:
Scrolling to reveal the instructions is good, but scrolling so far that the OS selection hides is not so good. It’s unexpected, and it hides the reassurance that you chose what you thought you did. It might be necessary in small viewports, or if the buttons are scrolled near the bottom of the viewport, but not otherwise. (@anasereijo independently had the same thought as me here.)

Could we instead scroll just enough to reveal, say, the top 3rem of the selected tab? And if at least that much is already visible, don’t scroll at all.